### PR TITLE
Add aws plugin

### PR DESCRIPTION
**Pull request by Hevienz**

### CODE REVIEW

File: api/cluster.proto [hunk 1]
```diff
@@ -20,17 +20,19 @@ message DescribeSubnetsRequest {
 	uint32 offset = 3;
 	google.protobuf.UInt32Value subnet_type = 4;
 	repeated string subnet_id = 5;
-	repeated string advanced_param = 6;
+	repeated string zone = 6;
```
Comment (chilianyi) [inline api/cluster.proto]: Is there zone in runtime？

File: deploy/config/global_config.init.yaml [hunk 2]
```diff
@@ -28,3 +29,8 @@ runtime:
     zone: .*
     image_id: xenial4x64a
     image_url: https://openpitrix.pek3a.qingstor.com/image/ubuntu.tar.gz
+  aws_provider:
+    api_server: ec2.us-east-2.amazonaws.com
+    zone: us-east-2
```
Comment (chilianyi) [inline deploy/config/global_config.init.yaml]: The image id is different between different zone?

File: api/cluster.proto (hunk 1 above)
Comment (Hevienz) [reply api/cluster.proto]: We need a field to determine whitch availabilityZone to deploy to. It just like we have pek3 region, then I must choice a availabilityZone to deploy my cluster.

File: deploy/config/global_config.init.yaml (hunk 2 above)
Comment (Hevienz) [reply deploy/config/global_config.init.yaml]: Yes.

**Pull request merged**